gce_image_import_export_tests/test_suites/image_import_export_test_suite.go

```diff
@@ -0,0 +1,238 @@
+	ctx context.Context, testCase *junitxml.TestCase,
+	logger *log.Logger, testProjectConfig *testconfig.Project) {
+
+	suffix := pathutils.RandString(5)
+	imageName := "e2e-test-image-import-os-from-image-" + suffix
+	cmd := "gce_vm_image_import"
+	args := []string{"-client_id=e2e", fmt.Sprintf("-project=%v", testProjectConfig.TestProjectID), fmt.Sprintf("-image_name=%v", imageName), "-os=debian-9", "-source_image=e2e-test-image-10g"}
+	runCliTool(logger, testCase, cmd, args)
+
+	verifyImportedImage(ctx, testCase, testProjectConfig, imageName, logger)
+}
+
+func runImageExportRawTest(
+	ctx context.Context, testCase *junitxml.TestCase,
+	logger *log.Logger, testProjectConfig *testconfig.Project) {
+
+	suffix := pathutils.RandString(5)
+	bucketName := fmt.Sprintf("%v-test-image", testProjectConfig.TestProjectID)
+	objectName := fmt.Sprintf("e2e-export-raw-test-%v", suffix)
+	fileURI := fmt.Sprintf("gs://%v/%v", bucketName, objectName)
+	cmd := "gce_vm_image_export"
+	args := []string{"-client_id=e2e", fmt.Sprintf("-project=%v", testProjectConfig.TestProjectID), "-source_image=e2e-test-image-10g", fmt.Sprintf("-destination_uri=%v", fileURI)}
+	runCliTool(logger, testCase, cmd, args)
+
+	verifyExportedImageFile(ctx, testCase, testProjectConfig, bucketName, objectName, logger)
+}
+
+func runImageExportVMDKTest(
+	ctx context.Context, testCase *junitxml.TestCase,
+	logger *log.Logger, testProjectConfig *testconfig.Project) {
+
+	suffix := pathutils.RandString(5)
+	bucketName := fmt.Sprintf("%v-test-image", testProjectConfig.TestProjectID)
+	objectName := fmt.Sprintf("e2e-export-vmdk-test-%v", suffix)
+	fileURI := fmt.Sprintf("gs://%v/%v", bucketName, objectName)
+	cmd := "gce_vm_image_export"
+	args := []string{"-client_id=e2e", fmt.Sprintf("-project=%v", testProjectConfig.TestProjectID), "-source_image=e2e-test-image-10g", fmt.Sprintf("-destination_uri=%v", fileURI), "-format=vmdk"}
+	runCliTool(logger, testCase, cmd, args)
+
+	verifyExportedImageFile(ctx, testCase, testProjectConfig, bucketName, objectName, logger)
+}
+
+func runCliTool(logger *log.Logger, testCase *junitxml.TestCase, cmdString string, args []string) {
+	// Execute cli tool
+	logger.Printf("[%v] Running command: '%s %s'", testCase.Name, cmdString, strings.Join(args, " "))
+	cmd := exec.Command(fmt.Sprintf("./%s", cmdString), args...)
+	cmd.Stdout = os.Stdout
+	cmd.Stderr = os.Stderr
+
+	if err := cmd.Run(); err != nil {
+		logger.Fatalf("Error running cmd: %v\n", err.Error())
+	}
+}
+
+func verifyImportedImage(ctx context.Context, testCase *junitxml.TestCase, testProjectConfig *testconfig.Project, imageName string, logger *log.Logger) {
+	logger.Printf("Verifying imported image...")
+	image, err := compute.CreateImageObject(ctx, testProjectConfig.TestProjectID, imageName)
+	if err != nil {
+		testCase.WriteFailure("Error creating compute api client: %v", err)
+		logger.Printf("Error creating compute api client: %v", err)
+		return
+	}
+
+	if err := image.Exists(); err != nil {
```

Comment:
Do we want to verify the content of the image as well?

Reply:
We can do it, or not do it. I considered about it, and I feel it's enough to only verify content in daisy workflow e2e tests (that has been implemented). In this test suites, it's fine to just verify whether daisy works as expected.

Reply:
Lets leave it as is. Later we can come back to implement more robust verification